ChangeLog.md

```diff
@@ -11,6 +11,11 @@
 * Make HasSource and HasSink superclasses of HasState.
   See [#75](https://github.com/tweag/capability/pull/75)
 
+* Allow to retain capabilities in zoom and magnify.
```

Comment:
```suggestion
* Handlers `zoom` and `magnify` can now carry capabilities over from the context.
```

(mostly because “allow to” is ungrammatical in English)